src/tools/pub/cmd/pub-get.md

```diff
@@ -145,6 +145,13 @@ By default, pub precompiles executables
 in immediate dependencies (`--precompile`).
 To prevent precompilation, use `--no-precompile`.
 
+### `--legacy-packages-file`
+
+If you utilize any third-party tools with a legacy dependency on `.packages` (desupported in [2.18](/guides/whats-new)),
+use `--legacy-packages-file` to generate a `.packages` file.
```

Comment:
I would rather not link to the "What's new" page as that is more about what is new on the website, not the Dart SDK itself. Support is being removed in Dart 2.19 so perhaps we can just include the version this is useful for as a main part of the sentence?

Perhaps something like:

```suggestion
If you are using Dart 2.18 and have any third-party tools
which still use the discontinued `.packages` file,
use `--legacy-packages-file` to generate a `.packages` file.
```